media-libs/kcolorpicker/kcolorpicker-0.3.0.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 2020-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+VIRTUALX_REQUIRED="test"
+inherit cmake multibuild virtualx
+
+MY_PN=kColorPicker
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Qt based color picker with popup menu"
+HOMEPAGE="https://github.com/ksnip/kColorPicker"
+SRC_URI="https://github.com/ksnip/${MY_PN}/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~loong ~ppc64 ~riscv ~x86"
+IUSE="+qt5 qt6 test"
+REQUIRED_USE="|| ( qt5 qt6 )"
+
+RDEPEND="
+	qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtwidgets:5[png]
+	)
+	qt6? (
+		dev-qt/qtbase:6[gui,widgets]
+	)
+"
+DEPEND="${RDEPEND}
+	test? (
+		qt5? ( dev-qt/qttest:5 )
+	)
+"
+
+S="${WORKDIR}/${MY_P}"
```

Comment:
We typically set this below `SRC_URI` these days, since subdir is usually linked with tarball name.